src/Controllers/DataService.cs

```diff
@@ -48,17 +48,10 @@ public static async Task<IActionResult> Read<T>(HttpRequest request)
 
             string path = RequestLogger.GetPathAndQuerystring(request);
 
-            CorrelationVector cVector = Middleware.CorrelationVectorExtensions.GetCorrelationVectorFromContext(request.HttpContext);
```

Comment:
This looks like we are sending Correlation Vector to CosmosDB in the HTTP header. Does this have any significance for tracking CosmosDB requests? We should confirm with Joseph or Bart.

Also, should we consider add B3-tracing in place of CV or removing is fine?

Reply:
opentelemetry autoinstrumentation is replacing the removed code